suite-native/module-accounts-import/src/components/AccountImportSummaryForm.tsx

```diff
@@ -25,7 +25,7 @@ import { HIDDEN_DEVICE_ID, HIDDEN_DEVICE_STATE, importAccountThunk } from '../ac
 import { AccountImportSummarySection } from './AccountImportSummarySection';
 
 type AccountImportSummaryFormProps = {
-    networkSymbol: NetworkSymbol;
+    currencySymbol: NetworkSymbol;
```

Comment:
Please do not use name `currencySymbol` it's confusing because it could either fiat or crypto, so you should prefix that. Better stick with just `networkSymbol` which is widely used in codebase and we should continue with that.

Reply:
b4d80e61f1d3d2f38840c1417c8cb8450de8156f